source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -264,9 +264,9 @@ Requires: Wire stripper, small flat screwdriver (optional), yellow/green twisted
 
 .. note: The PCM is an optional component used for controlling pneumatics on the robot. If you are not using the PCM, wire the CAN connection directly from the roboRIO (shown in the above step) to the PDP (show in this step).
```

Comment:
any chance you can fix the `show->shown` typo here?

Reply:
Fixed both there was one above it too! Thanks